### PR TITLE
elbv3 types

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Added types for ELB V3

### CODE REVIEW

File: octavia_proxy/api/drivers/elbv3/driver.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
-from oslo_log import log as logging
-
 from octavia_lib.api.drivers import provider_base as driver_base
+from oslo_log import log as logging
 
-from octavia_proxy.api.v2.types import load_balancer
 from octavia_proxy.api.v2.types import listener as _listener
-
+from octavia_proxy.api.v2.types import load_balancer
```
Comment (outcatcher) [inline octavia_proxy/api/drivers/elbv3/driver.py:5]: Those imports can be in a single line

File: octavia_proxy/api/v3/types/l7policy.py [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from wsme import types as wtypes
+
+from octavia_proxy.api.common import types
+from octavia_proxy.api.v3.types import l7rule
+from octavia_proxy.api.v3.types import listener
+from octavia_proxy.api.v3.types import pool
```
Comment (outcatcher) [inline octavia_proxy/api/v3/types/l7policy.py:20]: This can be in a single import line

File: octavia_proxy/api/v3/types/listener.py [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from dateutil import parser
+from oslo_config import cfg
+from wsme import types as wtypes
+
+from octavia_proxy.api.common import types
+from octavia_proxy.api.v3.types import l7policy
+from octavia_proxy.api.v3.types import pool
```
Comment (outcatcher) [inline octavia_proxy/api/v3/types/listener.py:20]: Can be a single line

File: octavia_proxy/api/v3/types/load_balancer.py [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+
+
+#    _child_map = {'vip': {
+#        'ip_address': 'vip_address',
+#        'subnet_id': 'vip_subnet_id',
+#        'port_id': 'vip_port_id',
+#        'network_id': 'vip_network_id',
+#        'qos_policy_id': 'vip_qos_policy_id',
+#        },
+#        'listeners': 'listeners'}
```
Comment (outcatcher) [inline octavia_proxy/api/v3/types/load_balancer.py:45]: Useless code?

File: octavia_proxy/api/v3/types/load_balancer.py [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        data_model.listeners = None
+        data_model.pools = None
+        result = super(LoadBalancerResponse, cls).from_data_model(
+            data_model, children=children)
+        #        if data_model.vip:
+        #            result.vip_subnet_id = data_model.vip.subnet_id
+        #            result.vip_port_id = data_model.vip.port_id
+        #            result.vip_address = data_model.vip.ip_address
+        #            result.vip_network_id = data_model.vip.network_id
+        #            result.vip_qos_policy_id = data_model.vip.qos_policy_id
```
Comment (outcatcher) [inline octavia_proxy/api/v3/types/load_balancer.py:100]: Not used

File: octavia_proxy/api/v3/types/pool.py [hunk 6]
```diff
@@ -0,0 +1,236 @@
...
+#    under the License.
+
+from octavia_lib.common import constants as lib_constants
+from wsme import types as wtypes
+
+from octavia_proxy.api.common import types
+from octavia_proxy.api.v3.types import health_monitor
+from octavia_proxy.api.v3.types import listener
+from octavia_proxy.api.v3.types import load_balancer
+from octavia_proxy.api.v3.types import member
```
Comment (outcatcher) [inline octavia_proxy/api/v3/types/pool.py:22]: Can be a one line

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: octavia_proxy/api/v3/types/load_balancer.py (hunk 4 above)
Comment (anton-sidelnikov) [reply octavia_proxy/api/v3/types/load_balancer.py]: yes)

**Review by outcatcher — APPROVED** [review]

**Pull request merged**